python/mxnet/numpy/linalg.py

```diff
@@ -890,9 +891,13 @@ def eigvalsh(a, UPLO='L'):
     >>> a = np.array([[ 5.4119368 ,  8.996273  , -5.086096  ],
     ...               [ 0.8866155 ,  1.7490431 , -4.6107802 ],
     ...               [-0.08034172,  4.4172044 ,  1.4528792 ]])
-    >>> LA.eigvalsh(a, UPLO='L')
+    >>> LA.eigvalsh(a, upper=False)
     array([-2.87381886,  5.10144682,  6.38623114]) # in ascending order
     """
+    if(upper==False):
```

Comment:
Comparison to `False` in python should be not expression: `if(upper==False)` => `if not upper`